dask/dataframe/tests/test_dataframe.py

```diff
@@ -469,7 +469,7 @@ def test_describe_empty():
         df_none.describe(), ddf_none.describe(percentiles_method="dask").compute()
     )
 
-    with pytest.raises(ValueError):
+    with pytest.raises((ValueError, RuntimeWarning)):
```

Comment:
Right - Good question.  We seem to be taking a different code path with fusion disabled, so we are running into a true-division RunTime error rather than a "No non-trivial arrays found" ValueError.  I'll poke around a bit before taking a stance on whether or not this is a problem :)

Reply:
I think this is the only lingering thing to address. Any insight into why we started raising a `RuntimeWarning`? I'm also happy to look into it